src/main/java/com/faforever/api/data/converter/MapParamsConverter.java

```diff
@@ -0,0 +1,50 @@
+package com.faforever.api.data.converter;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+import java.io.IOException;
+import java.util.Map;
+
+@Slf4j
+@Converter
+public class MapParamsConverter implements AttributeConverter<Map<String, Object>, String> {
+
+  private final ObjectMapper objectMapper;
+
+  public MapParamsConverter(ObjectMapper objectMapper) {
+    this.objectMapper = objectMapper;
+  }
+
+  @Override
+  public String convertToDatabaseColumn(Map<String, Object> mapParams) {
+
+    String mapParamsJson = null;
+    try {
+      mapParamsJson = objectMapper.writeValueAsString(mapParams);
+    } catch (final JsonProcessingException e) {
+      log.error("JSON writing error", e);
```

Comment:
"Failed to convert Json object to String"

Reply:
Done